CHANGELOG

```diff
@@ -96,6 +98,13 @@ New and updated examples and miniapps
   inhomogeneous), Robin, and periodic boundary conditions with either H1 or DG
   discretizations.
 
+- Ported example 11p to SLEPc, to demonstrate solving the Laplace eigenvalue
+  equation with the shift-and-invert spectral transformation method.
+
+- Added a new example, Example 28p, to demonstrate the solution of the 2D
+  Maxwell eigenvalue problem in a dielectric waveguide. This examples shows the
+  use of the SLEPc eigensolvers with mixed finite elements and block operators.
+
```

Comment:
Still mentions the soon to be miniapp (ex28p)

Reply:
Fixed